packages/anthropic/src/anthropic-api-types.ts

```diff
@@ -99,11 +99,11 @@ export type AnthropicTool =
     }
   | {
       name: string;
-      type: 'text_editor_20241022';
+      type: 'text_editor_20250124' | 'text_editor_20241022';
```

Comment:
Sort by newest to match the order for `computer_*` tools on [line 95 above](https://github.com/rmarescu/vercel-ai/blob/0407f218b06fbff059989fea4a09122840a541d7/packages/anthropic/src/anthropic-api-types.ts#L95).